synfig-studio/src/gui/dials/resolutiondial.cpp

```diff
@@ -54,11 +51,31 @@ using namespace studio;
 /* === P R O C E D U R E S ================================================= */
 
 /* === M E T H O D S ======================================================= */
+static void
+init_button(Gtk::ToolButton& button, const std::string& icon_name, const std::string& label, const std::string& tooltip)
+{
+	button.set_icon_name(icon_name);
+	button.set_label(label);
+	button.set_tooltip_text(tooltip);
+	button.show();
+}
 
-ResolutionDial::ResolutionDial(const Gtk::IconSize &size)
+static void
+init_toggle_button(Gtk::ToggleToolButton& button, const std::string& label, const std::string& tooltip)
 {
-	init_button(increase_resolution, size, Gtk::StockID("synfig-increase_resolution"), _("Increase Resolution"), _("Increase Display Resolution"));
-	init_button(decrease_resolution, size, Gtk::StockID("synfig-decrease_resolution"), _("Decrease Resolution"), _("Decrease Display Resolution"));
+	// For label left/right padding
+	button.set_name("low-resolution");
+
+	button.set_label(label);
+	button.set_tooltip_text(tooltip);
+	button.set_is_important(true);
+	button.show();
+}
```

Comment:
This should be in section `P R O C E D U R E S`

Reply:
Done